support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -197,111 +191,140 @@ function getServerSideParticipations(): Participations | null | undefined {
 	return null;
 }
 
-function getAmountsTestFromURL(data: AcquisitionABTest[]) {
+function getAmountsTestFromURL(
+	data: AcquisitionABTest[],
+): AcquisitionABTest | null {
 	const amountTests = data.filter((t) => t.testType === 'AMOUNTS_TEST');
 	if (amountTests.length) {
-		const test = amountTests[0];
-		return {
-			[test.name]: test.variant,
-		};
+		return amountTests[0];
 	}
 	return null;
 }
 
-function getAmountsTestParticipations(
+interface GetAmountsTestVariantResult {
+	selectedAmountsVariant: SelectedAmountsVariant; // Always return an AmountsVariant, even if it's a fallback
+	amountsParticipation?: Participations; // Optional because we only add participation if we want to track an amounts test that has multiple variants
+}
+export function getAmountsTestVariant(
 	country: IsoCountry,
 	countryGroupId: CountryGroupId,
 	settings: Settings,
-	acquisitionDataTests: AcquisitionABTest[],
-): Participations | null | undefined {
-	if (
-		!targetPageMatches(
-			window.location.pathname,
-			'/??/contribute|thankyou(/.*)?$',
-		)
-	) {
-		return null;
+	path: string = window.location.pathname,
+	mvt: number = getMvtId(),
+	acquisitionDataTests: AcquisitionABTest[] = getTestFromAcquisitionData() ??
+		[],
+): GetAmountsTestVariantResult {
+	const { amounts } = settings;
+	if (!amounts) {
+		return {
+			selectedAmountsVariant: getFallbackAmounts(countryGroupId),
+		};
 	}
 
-	// Amounts test/variant has been set in the Epic and coded into the Epic CTA URL
+	const buildParticipation = (
+		test: AmountsTest,
+		testName: string,
+		variantName: string,
+	): Participations | undefined => {
+		// Check if we actually want to track this test
+		const pathMatches = targetPageMatches(
+			path,
+			'/??/contribute|thankyou(/.*)?$',
+		);
+		if (pathMatches && test.variants.length > 1 && test.isLive) {
```

Comment:
we've noticed that, in the existing code, the check for more than one variant [only happens](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/abTests/abtest.ts#L296) if there isn't an amounts test in the url.
If there is an amounts test in the url then we currently do add it to the participations [regardless of the number of variants](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/abTests/abtest.ts#L229).

Reply:
This inconsistency is probably not deliberate, so we need to decide which is correct

Reply:
We've decided this logic is correct. Participation of an amounts test is only tracked if it's a genuine AB test with more than just a control